### PR TITLE
Add Trillium CI tests for 2.5 release

### PR DESCRIPTION
**Pull request by bhavya01**
# Description

Please include a summary of relevant context/issue and your changes.

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...

**List links for your tests (use go/shortn-gen for any internal link):** ...

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet [hunk 1]
```diff
@@ -34,7 +34,16 @@ local tpus = import 'templates/tpus.libsonnet';
...
+  trillium_4:: {
+    tpuSettings+: {
+      softwareVersion: 'v2-alpha-tpuv6e',
+    },
+    accelerator: tpus.trillium_4,
+  },
+
   configs: [
     ci + v5litepod_4 + pjrt,
+    ci + pjrt + trillium_4,
```
Comment (zpcore) [inline dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet:47]: Does the sequence matter here?

**Review by zpcore — APPROVED** [review]
LGTM!

File: dags/legacy_test/tests/pytorch/r2.5/common.libsonnet [hunk 2]
```diff
@@ -91,7 +91,12 @@ local volumes = import 'templates/volumes.libsonnet';
       tpuVmPytorchSetup: |||
         pip3 install -U 'setuptools>=70.0.0,<71.0.0'
         # `unattended-upgr` blocks us from installing apt dependencies
-        sudo systemctl stop unattended-upgrades
+        if systemctl is-active --quiet unattended-upgrades; then
+          sudo systemctl stop unattended-upgrades
+          echo "unattended-upgrades stopped."
+        else
+          echo "unattended-upgrades is not running."
+        fi
```
Comment (ManfeiBai) [inline dags/legacy_test/tests/pytorch/r2.5/common.libsonnet:99]: Thanks for this fix

File: dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet (hunk 1 above)
Comment (bhavya01) [reply dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet:47]: Yes, it matters. trillium_4 overrides the software version set by default in pjrt

**Review by ManfeiBai — APPROVED** [review]
Thanks, LGTM

File: dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet (hunk 1 above)
Comment (zpcore) [reply dags/legacy_test/tests/pytorch/r2.5/ci.libsonnet:47]: Oh, good to know! Thanks

**Pull request merged**